dist/split-test.js

```diff
@@ -17,6 +17,8 @@
     }
 }(this, function() {
 
+    var DEFAULT_LIFETIME = 30 * 24 * 3600;
```

Comment:
+1 to the way Don made the calculation - it makes it a bit more clear how to tell how long things are by breaking them down into units.